### ISSUE TITLE
Example app is not working

### ISSUE THREAD

**Issue by satazor** [ipfs-shipyard/peer-star-app#82]
The example app fails to start due to a html webpack plugin error. The culprit was the the merge of the greenkeeper commit that updated the dependencies.

Should we revert it or update to last CRA (2.0 is out)?

**Comment by pgte** [comment]
I'd prefer updating to latest :)

**Comment by satazor** [comment]
@pgte could you summarize the changes you made after ejecting so I can reapply?

**Comment by pgte** [comment]
I don't remember anymore, unfortunately I didn't commit before and after.. :(
I think that it had to do with fixing resolving a the peer-star-app dependency to a parent folder..
